.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Test
+
+on: ['push', 'pull_request']
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+      
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install Dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn
+
+      - name: Test
+        run: yarn test --ci
+        env:
+          COVERALLS_REPO_TOKEN: '${{ secrets.COVERALLS_REPO_TOKEN }}'
+
+      - name: Run coverage (with lcov file as output)
```

Comment:
maybe we should rename this script? Because it is not only collecting coverage but also runs the tests. I think we do not need the previous step, right? 🤔